bodhi/server/templates/update.html

```diff
@@ -650,6 +650,15 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              <div class="p-b-1">
+                <div>
+                  <strong>CI Status</strong>
+                </div>
+                <div>
+                  ${self.util.ci_status2html(update.ci_status_label) | n}
+                </div>
+              </div>
```

Comment:
I suggest renaming ```ci_status_label``` to ```ci_status``` and checking to see if it returns None or not before displaying the above ```<div>```. That way our existing Updates won't show CI Status since they will never have tests.